vendor/ddcloud/resource_additional_nics.go

```diff
@@ -31,11 +30,11 @@ func resourceAdditionalNic() *schema.Resource {
 				Required:    true,
 				Description: "ID of the server to which the additional nics needs to be updated",
 			},
-			resourceKeyNicID: &schema.Schema{
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "ID of the nic",
-			},
+			// resourceKeyNicID: &schema.Schema{
```

Comment:
BTW, if you're not still working on this, feel free to delete the commented-out code.